apps/backend/src/apikeys/apikey.service.ts

```diff
@@ -40,7 +40,13 @@ export class ApiKeyService {
     );
     // Since BCrypt has a 72 byte limit only hash the JWT signature
     const JWTSignature = newJWT.split('.')[2];
-    newApiKey.apiKey = await hash(JWTSignature, 14);
+    newApiKey.apiKey = await hashAndSaltPassword(
+      JWTSignature,
+      !(
+        this.configService.get('USE_NEW_ENCRYPTION_STRATEGY')?.toLowerCase() ===
```

Comment:
USE_NEW_ENCRYPTION_STRATEGY can be renamed, but is the concept of setting Heimdall to be FIPS enabled